recipes/noqt/meta.yaml

```diff
@@ -13,29 +13,18 @@ outputs:
   - name: noqt
     requirements:
       run:
-        - {{ pin_subpackage('noqt4', exact=True) }}
         - {{ pin_subpackage('noqt5', exact=True) }}
         - {{ pin_subpackage('noqt6', exact=True) }}
     test:
       commands:
         - echo "no tests for this package"
-  - name: noqt4
-    build:
-      noarch: generic
-    requirements:
-      run_constrained:
-        # qt4 only ever had a "mega qt" package called qt
-        - qt <0.0a0|>=5.0a0
-    test:
-      commands:
-        - echo "no tests for this package"
   - name: noqt5
     build:
       noarch: generic
     requirements:
       run_constrained:
         # qt-main only exists as of 5.x
-        - qt-main <0.0a0
+        - qt-main <5.0a0|>=6.0a0
```

Comment:
I remember some discussion in mamba recently about these kind of "or" specifiers. Can we get confirmation that they're fully supported in mamba (@jjerphan @JohanMabille) & pixi (@wolfv @baszalmstra)?

Reply:
Fully supported by rattler/py-rattler/pixi/rattler-build as well.